docs/kratos/concepts/index.md

```diff
@@ -211,3 +197,18 @@ integrates natively with ORY Oathkeeper, our Reverse Proxy, and defining Access
 of JSON / JSON5 / YAML!
 
 If you haven't seen the [Quickstart](../quickstart.md), we highly recommend checking that out!
+
+
+@aeneas
```

Comment:
Thanks, I've moved this to https://github.com/ory/web/blob/master/src/pages/blog/why-new-user-login-registration.md